repairnator/src/main/java/fr/inria/spirals/repairnator/process/inspectors/ProjectInspector.java

```diff
@@ -51,16 +52,15 @@
     private File[] repairSourceDir;
     private boolean isReproducedAsFail;
     private boolean isReproducedAsError;
-    protected Build previousBuild;
     protected boolean previousBuildFlag;
 
-    public ProjectInspector(Build failingBuild, String workspace, List<AbstractDataSerializer> serializers,
-            String nopolSolverPath, boolean push, RepairMode mode) {
-        this.build = failingBuild;
+    public ProjectInspector(BuildToBeInspected buildToBeInspected, String workspace, List<AbstractDataSerializer> serializers,
+                            String nopolSolverPath, boolean push, RepairMode mode) {
+        this.buildToBeInspected = buildToBeInspected;
```

Comment:
Actually you should also test in the run method that the buildToBeInspected status is the expected one